vocabularies-qsi/addr-geocode-types.ttl

```diff
@@ -11,203 +11,153 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
-    a skos:Concept ;
-    dcterms:provenance "G-NAF" ;
-    rdfs:isDefinedBy cs: ;
-    skos:altLabel "GCP"@en ;
-    skos:definition "Gas connection point (e.g. box, or underground chamber)."@en ;
-    skos:inScheme cs: ;
-    skos:prefLabel "Gas Connection Point"@en ;
-.
-
-:gas-meter
-    a skos:Concept ;
-    dcterms:provenance "G-NAF" ;
-    rdfs:isDefinedBy cs: ;
-    skos:altLabel "GM"@en ;
-    skos:definition "Gas meter point (e.g. box, or underground chamber)."@en ;
-    skos:inScheme cs: ;
-    skos:prefLabel "Gas Meter"@en ;
-.
-
-:internet-connection-point
-    a skos:Concept ;
-    dcterms:provenance "G-NAF" ;
-    rdfs:isDefinedBy cs: ;
-    skos:altLabel "ICP"@en ;
-    skos:definition "Internet connection point (e.g. box, or underground chamber)."@en ;
-    skos:inScheme cs: ;
-    skos:prefLabel "Internet Connection Point"@en ;
-.
-
 :letterbox
     a skos:Concept ;
-    dcterms:provenance "AS 4590.1:2017 | Standards Australia" ;
     rdfs:isDefinedBy cs: ;
     skos:altLabel "LB"@en ;
     skos:definition "Place where mail is deposited."@en ;
+    skos:historyNote "AS 4590.1:2017 | Standards Australia" ;
     skos:inScheme cs: ;
     skos:prefLabel "Letterbox"@en ;
     skos:topConceptOf cs: ;
 .
 
 :property-access-point
     a skos:Concept ;
-    dcterms:provenance "AS 4590.1:2017 | Standards Australia" ;
     rdfs:isDefinedBy cs: ;
     skos:altLabel "PAP"@en ;
     skos:definition "Access point (centre of) at the road frontage of the address site."@en ;
+    skos:historyNote "AS 4590.1:2017 | Standards Australia" ;
     skos:inScheme cs: ;
     skos:prefLabel "Property Access Point"@en ;
     skos:topConceptOf cs: ;
 .
 
 :property-access-point-setback
     a skos:Concept ;
-    dcterms:provenance "AS 4590.1:2017 | Standards Australia" ;
     rdfs:isDefinedBy cs: ;
     skos:altLabel "PAPS"@en ;
     skos:definition "Centre of driveway on address site frontage."@en ;
+    skos:historyNote "AS 4590.1:2017 | Standards Australia" ;
     skos:inScheme cs: ;
     skos:prefLabel "Property Access Point Setback"@en ;
     skos:topConceptOf cs: ;
 .
 
 :property-centroid
     a skos:Concept ;
-    dcterms:provenance "AS 4590.1:2017 | Standards Australia" ;
     rdfs:isDefinedBy cs: ;
     skos:altLabel "PC"@en ;
     skos:definition "Point of centre of parcels making up an address site and lying within its boundaries (e.g. for l-shaped address site)."@en ;
+    skos:historyNote "AS 4590.1:2017 | Standards Australia" ;
     skos:inScheme cs: ;
     skos:prefLabel "Property Centroid"@en ;
     skos:topConceptOf cs: ;
 .
 
 :service-connection-point
     a skos:Concept ;
-    dcterms:provenance "AS 4590.1:2017 | Standards Australia" ;
     rdfs:isDefinedBy cs: ;
     skos:altLabel "SCP"@en ;
     skos:definition "The utility connection point (e.g. box, or underground chamber)."@en ;
+    skos:historyNote "AS 4590.1:2017 | Standards Australia" ;
     skos:inScheme cs: ;
     skos:narrower
         :electricity-connection-point ,
```

Comment:
This should also include the other narrower items: 
water-connection-point
sewerage-connection-point
gas-connection-point
telephone-connection-point
electricity-connection-point
internet-connection-point

Reply:
Looks like they're there. You may need to click the blue portion in the GitHub interface to expand the file.